presentation/build.gradle.kts

```diff
@@ -89,6 +89,9 @@ dependencies {
     testImplementation(libs.compose.ui.test.junit)
     testImplementation(libs.android.test.espresso)
     testImplementation(libs.test.navigation)
+
+    // TODO: Find out how dependencies are being added
+    implementation("io.coil-kt:coil-compose:2.2.0")
```

Comment:
check out `[ReadMe.MD](https://github.com/droidconKE/droidconKE2022Android#readme)` under dependencies bit.

Reply:
Forgot to remove this 🤦‍♀️